user/reference/xenial.md

```diff
@@ -74,15 +74,9 @@ services:
 
 ## Environment common to all Xenial images
 
-In total, three different Xenial images are available for use. The selection
-process takes place based on the language and services specified in your `travis.yml` configuration.
-
-The minimal image `stevonnie` supports docker, bash, and has the tools for dynamic language runtime download and activation, for example, `rvm` for ruby.
-
-The other two images have different language runtimes installed by default, and
-have an array of different databases and services available.
-
-You can find below the environment common to all Xenial images. The environment specifications below are also the ones for our `language: minimal` Xenial image.
+As a technicallity, three different images have been created that all support a
```

Comment:
This has the same issue of mentioning images but not really explaining them. What do you think of:

The following versions of Docker, version control software and compilers are present on all builds, along with more language specific software described in more detail in each language section.